File: resources/[qb]/qb-core/shared/pollution.lua
QBShared = QBShared or {}

QBShared.Pollution = {}

QBShared.Pollution.Level = {
    Low = 1,
    Neutral = 2,
    High = 3
}

QBShared.Pollution.Threshold = {
    [QBShared.Pollution.Level.Low] = { min = 0, max = 11 },
    [QBShared.Pollution.Level.Neutral] = { min = 11, max = 70 },
    [QBShared.Pollution.Level.High] = { min = 70, max = 100 },
}

QBShared.Pollution.Multiplier = {
    [QBShared.Pollution.Level.Low] = 0.8,
    [QBShared.Pollution.Level.Neutral] = 1.0,
    [QBShared.Pollution.Level.High] = 1.25,
}

File: resources/[soz]/soz-core/src/server/pollution.ts
import { Injectable } from '../core/decorators/injectable';

export enum PollutionLevel {
    Low = 1,
    Neutral = 2,
    High = 3,
}

@Injectable()
export class Pollution {
    getPollutionLevel(): PollutionLevel {
        return exports['soz-upw'].GetPollutionLevel() as PollutionLevel;
    }
}
